flink-core/src/test/java/org/apache/flink/api/common/typeutils/CompositeSerializerTest.java

```diff
@@ -262,6 +263,7 @@ protected TestListCompositeSerializer createOuterSerializerWithNestedSerializers
         }
     }
 
+    @Ignore
```

Comment:
nit: maybe it's easier to grasp for future readers if we add an explanation?
```suggestion
        @Ignore("*TestInstance class is not a test")
```